config/application.rb

```diff
@@ -9,7 +9,9 @@ class Application < Rails::Application
     config.encoding = "utf-8"
     config.filter_parameters += [:password]
     config.active_support.escape_html_entities_in_json = true
-
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance| 
```

Comment:
I'm not really interested in handling my own errors at this point, but without this line, errors seemed to horribly screw up the styling on the page because of "field_with_error" or something like that, that rails was automatically adding.